### ISSUE TITLE
Api is not returning the link to the src of a jpg correctly

### ISSUE THREAD

**Issue by niposch** [bakapear/ifunny-web-api#1]
# How did it happen
```

var ifunny = require("ifunny-web-api");
ifunny({ shuffle: false }, callback);
function callback(err, res) {
    console.log(res[0].src);
}
 
```
# Output

res[0]:  

>     src:"/images/1x1.gif" // does not work  

>     tags:"when i was ..." // works  

>     type:"jpg"  // works

>     url:"https://ifunny.co/picture/H00lkjj47"  // works

# Expected behavior
In my oppinion **res[0].src** should be equal to a link to the image  ("https://img.ifunny.co/images/04577f768b03ccd2d8cb039d35945fe081737a8ff91ecb727b985009c2ef7fe4_1.jpg" in that case)

**Comment by niposch** [comment]
I have a fix for it. Not able to create branch+pull request though, so here is what to do
change line 30 in ifunny.js from:

`item["src"] = data[i].children[0].next.children[0].next.children[0].attribs.src`

to:
`item["src"] = data[i].children[0].next.children[0].next.children[0].attribs["data-src"]`

**Comment by bakapear** [comment]
Fixed in v1.0.1, thanks!
